.github/workflows/codeql-pr.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Analyze javascript
     runs-on: kibana
     if: |
-      (contains(fromJSON('["MEMBER", "COLLABORATOR"]'), github.event.pull_request.author_association) &&
+      (contains(fromJSON('["MEMBER", "COLLABORATOR", "CONTRIBUTOR"]'), github.event.pull_request.author_association) &&
```

Comment:
IIRC you have to be a `MEMBER` to add labels to PRs. Can we add a check to this part to see if at least one label exists on the PR, to prevent non-members from triggering this on their own?